roles/splunk/tasks/configure_idxc_manager.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Run splunk version command to check currently installed version
+  include_tasks: check_splunk_version.yml
+
+# Splunk version < 8.1 supports mode=master
+# https://docs.splunk.com/Documentation/Splunk/8.0.9/Indexer/Configuremasterwithserverconf
+# Splunk version >= 8.1 supports mode=manager
+# https://docs.splunk.com/Documentation/Splunk/latest/Indexer/Configuremanagerwithserverconf
+- name: Setting clustering mode based on Splunk version number
+  set_fact:
+    mode_value: {% if splunk_v_num_only < 8.1 %}'master'{% else %}'manager'{% endif %}
+
+- name: view mode_value
+  debug:
+    msg: "mode value is {{ mode_value }}"
```

Comment:
Remove this

Reply:
Oh shoot, thanks - forgot to remove after testing.